packages/node-resolve/src/index.js

```diff
@@ -136,6 +136,14 @@ function resolveImportSpecifiers(importSpecifierList, resolveOptions) {
   return p;
 }
 
+function getPackageName(id) {
+  const split = id.split('/');
+  if (split[0][0] === '@') {
+    return `${split[0]}/${split[1]}`;
```

Comment:
should this just be `id`?

Reply:
No, this will turn an import of `@my-scope/my-package/foo.js`  into `@my-scope/my-package`.

Reply:
@LarsDenBakker that's excellent info. could you add a code comment to that effect there?

Reply:
Yeah makes sense, I've added comments to the function.